priv/mime.types

```diff
@@ -509,7 +534,7 @@ application/vnd.antix.game-component
 application/vnd.apache.thrift.binary
 application/vnd.apache.thrift.compact
 application/vnd.apache.thrift.json
-application/vnd.api+json    json-api
+application/vnd.api+json
```

Comment:
```suggestion
application/vnd.api+json    json-api
```

Reply:
Can do, even if `json-api` is not a IANA valid type?

Reply:
Yeah, it is an addition we had.